src/solvers/dg.jl

```diff
@@ -12,6 +12,10 @@ function get_element_variables!(element_variables, u, mesh, equations,
     nothing
 end
 
+function get_element_variables!(element_variables, mesh)
```

Comment:
Maybe add a comment here why this is here?

Reply:
True. It took me a while to understand what these functions are for. In 8e08c79c, I added a comment for this function and also for the one that dispatches on the volume integral type.